scripts/apps/monitoring/controllers/MonitoringController.ts

```diff
@@ -145,8 +146,14 @@ export function MonitoringController($rootScope, $scope, $location, desks, super
         let groupLabel;
 
         if (group.subheader) {
-            groupLabel = activeWorkspace === 'workspace' ?
-                group.header + ' ' + group.subheader : group.subheader;
+            const stageLabelObject = getLabelForStageIfExists(group.subheader);
```

Comment:
see if you can translate the stage names earlier before they are converted into a `group`. `group` is not always equal to a stage. It can also be a saved search or maybe even a workspace group, not sure about that.

It'd be better to call this function only when we are sure that the object is a stage

Reply:
good catch